pages/contact-us/index.tsx

```diff
@@ -0,0 +1,178 @@
+import Link from 'next/link'
+import Heading from '../../components/Heading'
+import { getBetaPopupNotAvailableContent } from '../../graphql/mappers/beta-popup-page-not-available'
+import { getAuthModalsContent } from '../../graphql/mappers/auth-modals'
+import { GetContactUsContentReturnType, getContactUsContent } from '../../graphql/mappers/contact-us'
+import { getBetaBannerContent } from '../../graphql/mappers/beta-banner-opt-out'
+import { getBetaPopupExitContent } from '../../graphql/mappers/beta-popup-exit'
+import { AuthIsDisabled, AuthIsValid, Redirect } from '../../lib/auth'
+import { authOptions } from '../api/auth/[...nextauth]'
+import { getServerSession } from 'next-auth/next'
+// import ErrorPage from '../../components/ErrorPage'
```

Comment:
I'd rather we remove the commented out lines temporarily rather than leave them in if we're not using them. Can always see git history on the file if they need to be added back.

Reply:
done